search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -332,6 +333,113 @@ class AttributeInstanceServiceTests : WithTimescaleContainer {
             .hasSize(3)
     }
 
+    @Test
+    fun `it should return min value of all instances for 30 day`() = runTest {
```

Comment:
these tests are not really well named. what you are actually testing is that the buckets start / end where you are expecting them to start / end.